### ISSUE TITLE
write (or locate) a simple script that fails if not py3 compatible, to be run on travis build

### ISSUE THREAD

**Issue by bepetersn** [sc3/cookcountyjail#298]
I'm sure something exactly like this has been done before. My guess is that it's so simple that it hasn't been packaged, and so we might just put it together in a script ourselves. Might be worth researching to see if someone has done so still.

Specifically, I am looking to create a script to exit with a non-zero exit code if not python 3 compatible. I thought immediately of using python -3, but this just throws warnings. 

I want to include this in our travis build to make it fail if not py3 compatible, since this is a goal of ours.

**Comment by wilbertom** [comment]
Yeah would be cool but it would just make the tests always fail since 3rd party modules throw warnings.

**Comment by bepetersn** [comment]
Ah, which ones are those? But actually, Travis can easily be configured to
run different tests for different versions of Python.

The point is to keep in contact with our py3 compatibility. I have seen
(and written, mind you) some very non-py3compatible code. Print statements,
object inheriting, lots of very simple things.
